packages/core/utils/src/index.ts

```diff
@@ -298,3 +304,23 @@ export function isConstructor(obj) {
 export function isEs6Module(module) {
   return (typeof module === 'function' && isConstructor(module)) || (typeof module === 'object' && typeof module.default === 'function' && module.__esModule);
 }
+
+export function warnIfPackageNotDefinedLocally(packageName, warnFunc) {
+  const packageIsResolvable = findUp.sync("node_modules/" + packageName, {cwd: dappPath()});
+  if (!packageIsResolvable) {
+    return warnFunc("== WARNING: it seems "  + packageName + " is not defined in your dapp package.json; In future versions of embark this package should be a local dependency and configured as a plugin");
+  }
+
+  const dappPackage = fs.readJSONSync(dappPath("package.json"));
+  const { dependencies, devDependencies } = dappPackage;
+  if (!((dependencies && dependencies[packageName]) || (devDependencies && devDependencies[packageName]))) {
+    return warnFunc("== WARNING: it seems "  + packageName + " is not defined in your dapp package.json dependencies; In future versions of embark this package should be a local dependency and configured as a plugin");
+  }
+
+  const embarkConfig = fs.readJSONSync(dappPath("embark.json"));
+  if (!embarkConfig.plugins[packageName]) {
+    return warnFunc("== WARNING: it seems "  + packageName + " is not defined in your dapp embark.json plugins; In future versions of embark this package should be a local dependency and configured as a plugin");
```

Comment:
```suggestion
    return warnFunc("== WARNING: it seems "  + packageName + " is not defined in your dapp's embark.json plugins; In future versions of embark this package should be a local dependency and configured as a plugin");
```

Reply:
Might be worth to use a constant for that message since it is reused.
Might as well add a link to the Plugin docs too.
Also, bonus points if you use string templates instead (back-tick strings)